backend/app/assets/javascripts/spree/backend/handlebar_extensions.js

```diff
@@ -8,3 +8,10 @@ Handlebars.registerHelper('t', function (key) {
 Handlebars.registerHelper('edit_product_url', function (productId) {
   return Spree.routes.edit_product(productId)
 })
+Handlebars.registerHelper('name_or_presentation', function (optionTypePresentation, optionValue) {
+  if (optionTypePresentation === 'Color') {
```

Comment:
```suggestion
  if (optionTypePresentation === 'Color') {
```

let's compare name, not presentation if possible